pyanaconda/ui/gui/spokes/advstorage/dasd.py

```diff
@@ -42,7 +42,8 @@ class DASDDialog(GUIObject):
     def __init__(self, data, storage):
         GUIObject.__init__(self, data)
         self.storage = storage
-        self.dasd = self.storage.dasd
+        self.dasd = [d for d in self.storage.devices if d.type == "dasd"]
+        self.dasd.sort(key=lambda d: d.name)
```

Comment:
Yes, this is correct. We don't want to have a static list of DASDs (which it used to be) as well as we don't want to have a property for each device type (dasd, cdrom, disk, iscsi,...).